### ISSUE TITLE
TypeError: (0 , _sweetalert2.default) is not a function

### ISSUE THREAD

**Issue by micooz** [limonte/sweetalert2#385]
I am using webpack and I try to import the module(**v6.2.6**) via:

```
import {default as swal} from 'sweetalert2';
swal(...); // error
```

I also tried:

```
import swal from 'sweetalert2';
swal(...); // same error
```

I want to keep using ES6 style.

**Comment by toverux** [comment]
It is probably a build issue on our side, or yours.
Could you provide the minimal Webpack configuration to reproduce?

**Comment by limonte** [comment]
Fixed in [v6.2.7](https://github.com/limonte/sweetalert2/releases/tag/v6.2.7). Thanks a lot @micooz for reporting this issue!
